types/foundry-vtt-api/types/apps/sidebar/sidebar.d.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Render the Sidebar container, and after rendering insert Sidebar tabs
+ */
+declare class Sidebar extends Application {
+	/** Sidebar application instance */
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/apps/sidebar/sidebar.d.ts;
  Line: 5;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.